### ISSUE TITLE
Refresh content on mozilla.org/foundation

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/plan#378]
The Foundation's page is out of date: https://www.mozilla.org/en-US/foundation/ We need to refresh the content.

Driver: @edrushka
Copywriter: @kevzawacki
Design: Need to consult with MoCo first. @Roberto-Cruz has done some initial work based on current MoCo design. 
Final decision maker: Mark Surman via @HPaulJohnson

**Comment by KevZawacki** [comment]
Thanks for the edits, @edrushka. Accepted, and now ready for @HPaulJohnson.
Link: http://goo.gl/zkP1ke

**Comment by edrushka** [comment]
Update: MoCo is doing some redesign work on Moz.org so we'll see what they need before moving ahead with this project. There's a meeting Wed Mar 25. I'll update afterwards if there's news.

**Comment by edrushka** [comment]
Update: this is going to be a fairly large project on the MoCo side. Here's their deck: https://docs.google.com/a/mozillafoundation.org/presentation/d/1UR8cs6NUxkdEjZ-R3LGPWItgGtV-IjoE4RICrHirFhc/edit#slide=id.g395ee75c8_014

We're meeting again March 31 to dig into the Foundation content, which Kevin, Paul and I have already drafted.

**Comment by edrushka** [comment]
#### Timeline:
UX/IA/design prototyping: May 4 - June 30
Copy: May 11 - June 30
Web dev: July 1 - Sept 4
User testing: July 17 - July 30
L10N: Aug 12 - Sept 21
QA/Code review: Sept 7 - Sept 18 
Launch: Sept 21
Post-launch a/b test: Sept 22 - Oct

**Comment by edrushka** [comment]
Update: 
ZURB, the design firm contracted by MoCo to do this work, will provide us with a page template and we can tweak our existing updated copy [here] (https://docs.google.com/document/d/1yXbtmU7Cr3P9lR7MUm_p8389uVYDpkaFBY0B7pXVZdA/edit) to fit it. Paul will follow up with Jen Bertch to see what legal documents we need to continuing providing.

**Comment by edrushka** [comment]
@HPaulJohnson will ping Jen to see if we can get a copy-only update to the current page, since the redesign is going to take a while. Paul, please update when you hear back.

**Comment by edrushka** [comment]
Any word from Jen, @HPaulJohnson?

**Comment by HPaulJohnson** [comment]
We discussed in the context of the overall mozilla.org redesign. Given the timeline for that project she is okay with us moving forward with updating content on this page. @KevZawacki and I are revisiting copy from a few months ago, will update as appropriate and solicit feedback in this sprint.

**Comment by edrushka** [comment]
Any updates @HPaulJohnson?

**Comment by HPaulJohnson** [comment]
No progress. Should be able to tackle this next heartbeat.

**Comment by HPaulJohnson** [comment]
I talked to MoCo and the scope of these changes will be copy only for now. When copy is finalized I'll file a bug and an update to the page will be pretty quick.

@KevZawacki has drafted copy. I am reviewing now. Plan is to has signoff on copy by Friday and bug to be submitted.

**Comment by edrushka** [comment]
So close. Soooo close!
@KevZawacki has drafted copy. @HPaulJohnson has signed off. Just need to get the OK from ops and bug will be submitted.

**Comment by OpenMatt** [comment]
@edrushka status?

**Comment by edrushka** [comment]
Waiting for ops.

**Comment by benrito** [comment]
I would advocate for quickly refreshing this page and not worrying too much about the larger mozilla.org redesign—the content we'll need will be valuable now and in the future. 

Would also advocate for revisiting the copy, as our world has changed from April when it was written.

Perhaps we can open a new ticket for the next heartbeat, focused on a really simple pull request that doesn't spend too much time on design?

A stretch goal would be to include stuff from #567; will think on it as we do that work.

**Comment by HPaulJohnson** [comment]
Here's the most recent copy that Kevin is working to get approved for an update to make sure we are looking at : https://docs.google.com/document/d/1yXbtmU7Cr3P9lR7MUm_p8389uVYDpkaFBY0B7pXVZdA/edit 

As for design, I'm not sure we have much/if any room for design changes to that page as it would require support from MoCo. 

The updates to this page have been blocked (for various reasons) for a very long time so I'd like to move forward quickly  with this ticket to update copy (feel free to add your thoughts to the doc above). We are close to unblocking this small piece at least. If you need something richer or more designed in the short term we should think about where else we might host it and work that as a separate issue.

**Comment by KevZawacki** [comment]
I'm speaking with Mark this Friday about reshaping the [draft copy](https://docs.google.com/document/d/1yXbtmU7Cr3P9lR7MUm_p8389uVYDpkaFBY0B7pXVZdA/edit) and getting it ready for mozilla.org/foundation.

**Comment by HPaulJohnson** [comment]
@KevZawacki any update on this?

**Comment by KevZawacki** [comment]
@HPaulJohnson Finishing up revisions based on Mark's feedback -- will have a next draft ready to share later today.

**Comment by OpenMatt** [comment]
thanks guys

**Comment by KevZawacki** [comment]
I've revised our copy for mozilla.org/foundation per my conversation with Mark. Feel free to take a look and share feedback before we reshare with ops. (Please used comments or 'suggest' mode.)

https://docs.google.com/document/d/1qPNmBuR3RqAjKBcypmL_uVq8Q1xIBpMbj5Oov05Vpz8/edit

We don't have much liberty in terms of structure -- Mark was specific about the flow -- but if anything seems off or missing, please let me know.

**Comment by edrushka** [comment]
Thanks for continuing to work on this, @KevZawacki! I made a few comments in the doc. I think we should surface the "non-profit" element a little sooner.

**Comment by KevZawacki** [comment]
Thanks, @edrushka and @HPaulJohnson. I've just revised the copy based on your suggestions. 

Last piece of the puzzle is the "grants" section. I'm waiting on copy from Lynn, who says the current page needs a complete overhaul.

**Comment by edrushka** [comment]
In the interest of getting the copy updated, could we simply put "coming soon" in the grants section? A 'complete overhaul' sounds rather time-consuming, and it has already been months since Mitchell requested this page be updated.

**Comment by benrito** [comment]
I have content that we can use for the grants section. We could include in the following heartbeat.

**Comment by KevZawacki** [comment]
@benrito Great. Do we want to ship the [existing copy](https://docs.google.com/document/d/1qPNmBuR3RqAjKBcypmL_uVq8Q1xIBpMbj5Oov05Vpz8/edit) to Mark for approval today?

**Comment by edrushka** [comment]
Update: undergoing a legal review from Abigail.

**Comment by KevZawacki** [comment]
@edrushka @HPaulJohnson update from Angela & Abigail on 10/19:

*Hi Kevin,*

*Sorry for the delay in connecting you with Abigail!*

*Abigail: Kevin has organized an update of the Foundation page on mozilla.org. We’d like to get the text checked by legal, in particular in relation to the advocacy piece. It may be helpful to get Michael Durham to look at that as well regarding tax consequences of that text specifically. Would you be able to help with this?*

**Comment by KevZawacki** [comment]
@edrushka @HPaulJohnson 

We have a response from Abigail re: our [refreshed Foundation copy](https://docs.google.com/document/d/1qPNmBuR3RqAjKBcypmL_uVq8Q1xIBpMbj5Oov05Vpz8/edit). She writes:

>Hi Kevin,

>A few small comments below, but I think this looks great. I will connect with Michael Durham on the advocacy piece and send you any follow-up for that area.

Her comments are below, along with my responses; I've updated the copy accordingly.

>I assume all the facts are correct (e.g., tens of thousands of global volunteers, corporate structure & spending, etc.)

We should be all set -- most of the Ops team has given the greenlight. However, I will change "tens of thousands" to "thousands," since it's a number that's always in flux.  

>I’ve been told that Thunderbird is no longer really supported?

It's a bit ambiguous. So, I've swapped out "Thunderbird" for "other open source tools."

>Do references in the doc to “Mozilla” mean just the Foundation or MoFo and MoCo together? Might be worth clarifying this (but it's also very possible I'm the only one confused).

It might very well be confusing! Let me take a closer look

>Sentence at end of Para 3: “As a nonprofit, much of Mozilla’s work in the education space is made possible through your donations and the efforts of volunteers.” Are donations largely used for educational efforts and not for product-building or advocacy work? Unless that’s the case, I might move this sentence to the last paragraph on this page, just before “You can join us and help protect the Web...”

Correct: donations are largely used for education, so this sentence is relevant to this paragraph.

...

So, we're waiting on Michael. In the meantime, feel free to comment on Abigail's questions/my responses.

**Comment by KevZawacki** [comment]
@edrushka @HPaulJohnson 

Abigail and Michael have both signed off! Do we know who's able to swap in the [new copy](https://docs.google.com/document/d/1qPNmBuR3RqAjKBcypmL_uVq8Q1xIBpMbj5Oov05Vpz8/edit)? 

I'd like to send an email to Ops sayings, "This copy has been approved by legal and will be going live Tuesday, Nov. 3. If you have any last-minute thoughts, please let us know."

**Comment by edrushka** [comment]
I believe Jen Bertsch @jbertsch can swap the copy. Her team isn't on GH though so we'll have to file a bug. I can do that if you like.

**Comment by HPaulJohnson** [comment]
yes, we need to file this in bugzilla to get it updated. exact timing will be dependent on what else is in their queue. 

we are so close!!

**Comment by KevZawacki** [comment]
@edrushka Thanks! Let's do it, and ask that they wait until Monday, Nov. 2. That gives Ops a couple days for a final review.

**Comment by HPaulJohnson** [comment]
in progress and page updates expected to be made by next week

**Comment by hannahkane** [comment]
@HPaulJohnson - can we close or move to a different milestone?

**Comment by HPaulJohnson** [comment]
Went live this morning! Check it out at mozilla.org/foundation
